app/scripts/components/exploration/components/dataset-selector-modal/index.tsx

```diff
@@ -208,21 +109,30 @@ export function DatasetSelectorModal(props: DatasetSelectorModalProps) {
       revealed={revealed}
       onCloseClick={close}
       renderHeadline={() => (
-        <RenderModalHeader defaultSelect={defaultSelectFilter} />
+        <RenderModalHeader />
       )}
       content={
-        <ModalContentRender 
-          search={search} 
-          selectedIds={selectedIds} 
-          displayDatasets={datasetsToDisplay} 
-          onCheck={onCheck}
-        /> 
+        <CatalogContent
```

Comment:
😎 that we can pass the Catalog Content directly here. Can we adjust the styles so the filter component and selected pills component can align? (To be fair, they don't perfectly align on Catalog page either but it looks more obvious in a smaller modal)

![Screen Shot 2024-06-05 at 10 23 09 AM](https://github.com/NASA-IMPACT/veda-ui/assets/4583806/00de3f9d-0f00-4e79-8ce0-75149c1497bb)